bucket/tableplus.json

```diff
@@ -0,0 +1,31 @@
+{
+    "version": "110",
+    "description": "Relational database explorer",
+    "homepage": "https://tableplus.com",
+    "license": "Proprietary",
+    "url": "https://s3.amazonaws.com/tableplus-win-builds/110/TablePlusSetup.exe",
+    "hash": "be696fb0c6f1f39c51cae409e02a364ebcda0172c21cb42c941430f8a9eedce5",
+    "innosetup": true,
+    "architecture" :{
+        "32bit": {
+            "pre_install": "Get-ChildItem \"$dir\" '*,1*' | Rename-Item -NewName { $_.Name -Replace ',1' }"
```

Comment:
You need fullname due to compatiblity gap between PW5 and PWSH

Reply:
> You most likely need fullname due to compatiblity gap between PW5 and PWSH

Didn't know that.  Thanks for noticing!